payment_method.go

```diff
@@ -0,0 +1,125 @@
+	ChannelProperties    directdebit.ChannelProperties `json:"channel_properties"`
+	Type                 directdebit.Type              `json:"type"`
+	BankAccount          *directdebit.BankAccount      `json:"bank_account"`
+	DebitCard            *directdebit.DebitCard        `json:"debit_card"`
+}
+
+type OverTheCounterMethod struct {
+	Amount            float64                          `json:"amount"`
+	Currency          constant.CurrencyEnum            `json:"currency"`
+	ChannelCode       overthecounter.ChannelCode       `json:"channel_code"`
+	ChannelProperties overthecounter.ChannelProperties `json:"channel_properties"`
+}
+
+type VirtualAccountMethod struct {
+	Amount            float64                          `json:"amount"`
+	MinimumAmount     float64                          `json:"min_amount,omitempty"`
+	MaximumAmount     float64                          `json:"max_amount,omitempty"`
+	Currency          constant.CurrencyEnum            `json:"currency"`
+	ChannelCode       virtualaccount.ChannelCode       `json:"channel_code"`
+	ChannelProperties virtualaccount.ChannelProperties `json:"channel_properties"`
+}
+
+type QRCodeMethod struct {
+	Amount            float64                  `json:"amount"`
+	Currency          constant.CurrencyEnum    `json:"currency"`
+	ChannelCode       qrcode.ChannelCode       `json:"channel_code"`
+	ChannelProperties qrcode.ChannelProperties `json:"channel_properties"`
+}
+
+type PaymentMethodResponse struct {
+	ID                 string                           `json:"id"`
+	Type               constant.PaymentMethodTypeEnum   `json:"type"`
+	Country            constant.CountryEnum             `json:"country"`
+	MerchantID         string                           `json:"business_id"`
+	CustomerID         *string                          `json:"customer_id"`
+	ReferenceID        string                           `json:"reference_id,omitempty"`
+	Reusability        constant.ReusabilityEnum         `json:"reusability"`
+	Status             constant.PaymentMethodStatusEnum `json:"status"`
+	Actions            PaymentMethodActions             `json:"actions"`
+	Description        *string                          `json:"description"`
+	CreatedAt          time.Time                        `json:"created"`
+	UpdatedAt          time.Time                        `json:"updated"`
+	Metadata           *Metadata                        `json:"metadata"`
+	BillingInformation *BillingInformation              `json:"billing_information"`
+	FailureCode        *string                          `json:"failure_code"`
+
+	Ewallet            *EwalletMethod            `json:"ewallet"`
+	DirectBankTransfer *DirectBankTransferMethod `json:"direct_bank_transfer"`
+	DirectDebit        *DirectDebitMethod        `json:"direct_debit"`
+	Card               *card.Method              `json:"card"`
+	Cryptocurrency     *cryptocurrency.Method    `json:"cryptocurrency,omitempty"`
```

Comment:
can remove also since this is not live yet

Reply:
removed cryptocurrency (and related folders)